src/nspawn/nspawn.c

```diff
@@ -1219,7 +1219,14 @@ static int setup_timezone(const char *dest) {
         /* Fix the timezone, if possible */
         r = readlink_malloc("/etc/localtime", &p);
         if (r < 0) {
-                log_warning("/etc/localtime is not a symlink, not updating container timezone.");
+                log_warning("/etc/localtime (on system, not in container)
+                is not a symlink, not updating container timezone.");
```

Comment:
This line break is invalid and breaks the build. Also, how about trimming the text to "The host's /etc/localtime is not ..."?